shared/structures/src/members/PlatformMember.ts

```diff
@@ -485,7 +484,7 @@ export class PlatformMember implements ObjectWithRecords {
         return def.isEnabled(this)
     }
 
-    isPropertyEnabled(property: 'birthDay'|'gender'|'address'|'parents'|'emailAddress'|'phone'|'emergencyContacts'|'dataPermission'|'financialSupport', options?: {checkPermissions?: {user: UserWithMembers, level: PermissionLevel}}) {
+    isPropertyEnabled(property: 'birthDay'|'gender'|'address'|'parents'|'emailAddress'|'phone'|'emergencyContacts'|'dataPermission'|'financialSupport'|'uitpasNumber', options?: {checkPermissions?: {user: UserWithMembers, level: PermissionLevel}}) {
```

Comment:
Hier moet nog permission check komen -> niet iedereen kan uitpasnummer zien of bewerken